node/commons/compatibility-layer.ts

```diff
@@ -583,7 +583,7 @@ export const buildAttributePath = (selectedFacets: SelectedFacet[]) => {
       }
 
       return facet.key !== 'ft'
-        ? `${attributePath}${facet.key}/${removeDiacriticsFromURL(facet.value).replace(/ |%20/, '-')}/`
+        ? `${attributePath}${encodeURI(facet.key)}/${removeDiacriticsFromURL(facet.value).replace(/ |%20/, '-')}/`
```

Comment:
I would recommend encoding the `facet.value` too 🤔

Reply:
I accepted the suggestion but the values were already encoded wouldn't it encode it twice, it worked on the Workspace tho.

Reply:
Accepted the suggestion and created a new const to properly encode the URI and avoid duplicates